### ISSUE TITLE
deleting plugins doesnt remove the files

### ISSUE THREAD

**Issue by bryant-the-coder** [wbthomason/packer.nvim#725]
is it normal that when i remove a plugin. The file (empty) is still there? kinda bugging me. Windows 10 btw

**Comment by wbthomason** [comment]
I assume you've deleted them by running `PackerClean`, and confirming? Assuming so, this would probably be an upstream bug with `vim.fn.delete`, since we remove plugins in the `clean` operation by calling `vim.fn.delete(plugin_path, 'rf')`.

**Comment by bryant-the-coder** [comment]
Nope. I use `PackerSync`. It always prompt me to confirm if I wanna delete or not. If i say yes, the content of the file is deleted. and not the folder itself.

**Comment by wbthomason** [comment]
`PackerSync` runs `PackerClean` as a sub-operation. Can you please try manually deleting some directory with `vim.fn.delete(<YOUR PATH>, 'rf')`? We might be encountering the issue with `delete` noted in the help docs, specifically "Note: on MS-Windows it is not possible to delete a directory that is being used.", but I wouldn't have thought that a plugin directory you don't separately have open would count as "used".

**Comment by bryant-the-coder** [comment]
i will try that and i will further inform thanks.
